tests/linearizability/client.go

```diff
@@ -32,13 +33,17 @@ type recordingClient struct {
 	baseTime time.Time
 }
 
-func NewClient(endpoints []string, ids identity.Provider, baseTime time.Time) (*recordingClient, error) {
-	cc, err := clientv3.New(clientv3.Config{
+func NewClient(endpoints []string, ids identity.Provider, baseTime time.Time, opts ...config.ClientOption) (*recordingClient, error) {
```

Comment:
Why you add this, I don't see this used anywhere?

Reply:
It will be used to add user and password options.